Hooks.php

```diff
@@ -38,6 +38,12 @@ public static function onBeforePageDisplay(OutputPage &$out, Skin &$skin) {
       $event = Event::getInstance();
       $event->render($out);
     }
+      
+    if($wgGoogleRichCardsAnnotateBooks) {
+      $book = Event::getInstance();
```

Comment:
`Book::getInstance()`

Reply:
At least it does not lead to a crash, but then, i obtain a event instead of a book…